src/rust/vendor-rust-std.sh

```diff
@@ -0,0 +1,12 @@
+RUST_DIR="$(rustc --print=sysroot)"
+
+# The following two commands must be run as root in the docker container.
+# Copy the Cargo.lock for Rust to place the `cargo vendor` will see
+# cp "$RUST_DIR/lib/rustlib/src/rust/Cargo.lock" "$RUST_DIR/lib/rustlib/src/rust/library/test/"
+# chmod 777 $RUST_DIR/lib/rustlib/src/rust/library/test/Cargo.lock
```

Comment:
I am not sure what this means. I just ran this script inside docker and it seemed to work.

Reply:
Ok I see now that these are needed, but:

why not uncomment and run them automatically?

And could you add some more comments about why this is needed, and why the library/test/ directory, etc.? Maybe some reference? A bit hard to understand this.

Reply:
I moved the commands to the dockerfile